src/usr/share/batwatch/batwatch.py

```diff
@@ -53,124 +46,150 @@ def __eq__(self, other):
         return self.__dict__ == other.__dict__
 
     def __ne__(self, other):
-        return not self.__eq__(other)
+        return not self == other
 
+    def __gt__(self, other):
+        is_greater_than = False
+        if self.battery_count > other.battery_count:
+            is_greater_than = True
 
-class Batwatch(object):
-    """
-    Monitors the status of system battery and notifies designated recipient(s) of changes
-    by gpgmailer encrypted email.
-    """
+        elif self.battery_count == other.battery_count:
+            if self.charge_status > other.charge_status:
+                is_greater_than = True
 
-    __name__ = 'Batwatch'
+        return is_greater_than
 
-    def __init__(self, config):
-        """
-        Start Batwatch with its configuration, logger, and reference to the UPower bus.
+    def __lt__(self, other):
+        if self != other:
+            return not self > other
 
-        config: Object containing Batwatch configuration details.
-        """
-        self.logger = logging.getLogger(self.__name__)
+        return False
```

Comment:
Multiple returns? What is this? A dynamic language?

Reply:
You want me to make an entire variable here when the entire method is two lines?

Reply:
I would, unless it makes the code really ugly. That said, in some contexts Python seems to really favor multiple returns, so I break that rule myself sometimes.